### PR TITLE
[WIP] testing PR events

### PR DESCRIPTION
**Pull request by davemeyer**

### CODE REVIEW

File: consumers/infrastructure/stream-consumer/templates/deployment-sendNudge.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
```
Comment (andycmaj) [inline consumers/infrastructure/stream-consumer/templates/deployment-sendNudge.yaml]: these can all be created with a single deployment wrapped with a `range` that generates multiple versions of the deployment

* https://github.com/helm/helm/blob/master/docs/chart_template_guide/control_structures.md#looping-with-the-range-action
* https://github.com/helm/helm/issues/1311

```
# values.yaml
consumers:
  - topic: raw_activity
    handler: enrichRawActivity
  - topic: botany_activity
    handler: persistBotanyActivity
```

```
# deployment.yaml
...
{{- $consumer := .Values.consumers }}
---
# entire deployment inside range
...
metadata:
  name: {{ include "stream-consumer.fullname" . }}-{{ $handler }}
...
    containers:
        ...
        args: ["--topic", "{{ $topic }}", "--handler", "{{ $handler }}"]
{{- end }}
```

File: consumers/src/handlers/sendNudges/index.ts [hunk 2]
```diff
@@ -36,11 +36,10 @@ const handler: TopicMessageHandler<Nudge> = async (nudge: Nudge) => {
   })) as SlackImResponse;
 
   debug('consumers:sendNudges', 'imResponse', imResponse);
-
   try {
     await slack.chat.postMessage({
       ...nudge.message,
-      text: null,
+      text: 'hello',
```
Comment (andycmaj) [inline consumers/src/handlers/sendNudges/index.ts]: slack requires this?

Comment (davemeyer) [reply consumers/src/handlers/sendNudges/index.ts]: this is the 'notification' text. so it should probably be filled with a short version of the full message

Comment (andycmaj) [reply consumers/src/handlers/sendNudges/index.ts]: https://github.com/asynchronous-dev/react-chat-renderer/issues/10

**Pull request closed without merging**